test/expected_output/libvirt/kvm/box.xml

```diff
@@ -18,7 +18,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/bin/qemu-system-x86_64</emulator>
+    <emulator>/usr/local/bin/qemu-system-x86_64</emulator>
```

Comment:
I'm not keen on changing this. Debian and Fedora (and thus presumably the many distributions based on them) places qemu-system-x86_64 in /usr/bin. Technically Fedora puts it in /bin, but since /usr/bin is symlinked to /bin it's available there.

https://packages.debian.org/jessie/qemu-system-x86
https://apps.fedoraproject.org/packages/qemu-system-x86/